pkg/web/api.go

```diff
@@ -32,6 +33,9 @@ type API struct {
 
 	// Filters contains the registered filters
 	Filters []Filter
+
+	// Registry is the health indicators registry for this API
+	health.Registry
```

Comment:
would it make sense to add a little segement to the documentation (extensions) to specify the possiblity to add health indicators?